### ISSUE TITLE
Cat Dog Estimator might be overfitting

### ISSUE THREAD

**Issue by hereismari** [random-forests/tensorflow-workshop#27]
[Cat Dog Estimator](https://github.com/random-forests/tensorflow-workshop/tree/master/extras/cat_dog_estimator) appears to be overfitting, and data could be separated between training and test set for better evaluation.
